packages/lab/src/dropdown-next/DropdownNext.css

```diff
@@ -0,0 +1,107 @@
+.saltDropdownNext-button {
+  --dropdownNext-borderColor: var(--salt-editable-borderColor);
+  --dropdownNext-borderStyle: var(--salt-editable-borderStyle);
+  --dropdownNext-borderWidth: var(--salt-size-border);
+
+  align-items: center;
+  background: var(--saltDropdownNext-background, var(--dropdownNext-background));
+  color: var(--saltDropdownNext-color, var(--salt-text-primary-foreground));
+  display: flex;
+  font-family: var(--salt-text-fontFamily);
+  font-size: var(--saltDropdownNext-fontSize, var(--salt-text-fontSize));
+  height: var(--saltDropdownNext-height, var(--salt-size-base));
+  line-height: var(--saltDropdownNext-lineHeight, var(--salt-text-lineHeight));
+  min-height: var(--saltDropdownNext-minHeight, var(--salt-size-base));
+  min-width: var(--saltDropdownNext-minWidth, 160px);
+  padding-left: var(--saltDropdownNext-paddingLeft, var(--salt-spacing-100));
+  padding-right: var(--saltDropdownNext-paddingRight, var(--salt-spacing-100));
+  position: relative;
+  border: none;
+  border-bottom: var(--dropdownNext-borderWidth) var(--dropdownNext-borderStyle) var(--dropdownNext-borderColor);
+  justify-content: space-between;
+}
+
+.saltDropdownNext-button:hover {
+  --dropdownNext-borderColor: var(--salt-editable-borderColor-hover);
+
+  cursor: var(--salt-selectable-cursor-hover);
+}
+
+.saltDropdownNext-button:focus,
+.saltDropdownNext-button:active {
+  --dropdownNext-borderColor: var(--salt-editable-borderColor-active);
+  --dropdownNext-borderWidth: var(--salt-editable-borderWidth-active);
+
+  outline: var(--saltDropdownNext-outline, var(--salt-focused-outlineWidth) var(--salt-focused-outlineStyle) var(--salt-focused-outlineColor));
+}
+
+.saltDropdownNext-buttonText {
+  text-align: left;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  padding-right: var(--salt-spacing-100);
+}
+
+.saltDropdownNext-list {
+  --saltDropdownNext-itemCount: 5;
```

Comment:
should be a local variable --dropdownNext, but considering it's not redefined as far as i can see, can just use it directly, or at least just use fallback of 5 with the api there